server/graphql/v2/input/CommentReactionCreateInput.js

```diff
@@ -0,0 +1,32 @@
+import { GraphQLInputObjectType, GraphQLInt, GraphQLString } from 'graphql';
+
+import { ExpenseReferenceInput } from './ExpenseReferenceInput';
+
+/**
+ * Input type to use as the type for the comment input in createComment mutation.
+ */
+export const CommentReactionCreateInput = new GraphQLInputObjectType({
+  name: 'CommentReactionCreateInput',
+  fields: () => ({
+    emoji: {
+      type: GraphQLString,
+      description: 'The emoji associated with the reaction',
+    },
+    UserId: {
+      type: GraphQLString,
+      description: 'An unique identifier for the user id associated with this comment reaction',
+    },
+    CommentId: {
+      type: GraphQLString,
+      description: 'An unique identifier for the comment id associated with this comment reaction',
+    },
```

Comment:
This should be a reference, similar to what we do for `server/graphql/v2/input/ConnectedAccountReferenceInput.ts`

```suggestion
    comment: {
      type: CommentReferenceInput,
      description: 'An unique identifier for the comment id associated with this comment reaction',
    },
```